contents/docs/feature-flags/bootstrapping-and-local-evaluation.mdx

```diff
@@ -5,6 +5,12 @@ availability:
     selfServe: full
     enterprise: full
 ---
+
+PostHog offers two main ways to use feature flags. 
+
+- You can send a `decide` request to the PostHog API to resolve a flag, or 
```

Comment:
We call it decide request, but I think this will be more clearer to non-posthog people

```suggestion
- You can send a request to the PostHog `/decide` API endpoint to resolve a flag, or 
```